Apsitvarkom.DataAccess/Geocoder.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Apsitvarkom.Models;
+using System.Net.Http.Headers;
+using Newtonsoft.Json;
+
+
+namespace Apsitvarkom.DataAccess
+{
+    public class Geocoder : IGeocoder
+    {
+        private static readonly HttpClient client = new HttpClient();
+
+        public static string geocoderApiKey;
```

Comment:
I believe this is not the right way to do it.

Have a constructor, where you would take in the geocoderApiKey. I would also create the HttpClient in the constructor instead of its current location.